### ISSUE TITLE
v2.0.0-beta.14 - Nested Text does not work

### ISSUE THREAD

**Issue by JacobFischer** [diegomura/react-pdf#990]
**Describe the bug**

Nested `<Text>` and `<Link>` components throw with:

```
TypeError: Cannot read property 'font' of undefined
    at eval (ignoreChars.js?a52e:18)
    at Array.map (<anonymous>)
    at ignoreChars (ignoreChars.js?a52e:17)
    at getFragments (getAttributedString.js?52fe:107)
    at eval (_pipe.js?47c6:3)
    at eval (_arity.js?2550:14)
    at eval (_curryN.js?551a:34)
    at eval (_arity.js?2550:10)
    at eval (converge.js?9119:41)
    at _map (_map.js?45e9:6)
    at eval (converge.js?9119:40)
    ...
```

**To Reproduce**

Try to build a PDF with nested Text or Link components:

``` jsx
<Text>
  <Text>The following is a link </Text>
  <Link href="http://github.com/">to GitHub</Link>
</Text>
```

**Expected behavior**

Nested Text and Link components should work.

They did in `2.0.0-beta.6`, however I bumped versions and this bug appeared. Not sure which version between that and 14 introduced the bug.

**Desktop:**
 - OS: MacOS and Windows
 - Browse: Chrome
 - React-pdf version: 2.0.0-beta.14

**Other notes**

This is probably the same bug as in #976, however that issue was closed without a true resolution so I've opened up this issue to be more clear what the error is.

Additionally the `Link` component in that example will slice off the `h` in `http...`, but that bug is already reported in #980.

**Comment by asgerhallas** [comment]
I believe it's related to this: https://github.com/diegomura/react-pdf/pull/617#issuecomment-670658130

**Comment by ryancarville** [comment]
I am having the exact same error.  Anyone find a fix?  Thank you!

**Comment by diegomura** [comment]
Fixed in latest 2.0 versions. Just tested it
